extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js

```diff
@@ -85,13 +86,13 @@ RdfSkeletonListDialog.prototype._constructBody = function(body) {
     for (var i = 0; i < data.length; i++) {
       var element = data[i];
       var option = $('<option>').attr('id', i).text(element.name);
-      
+
       option.click(function(evt) {
         evt.preventDefault();
         $(".metadatapreview").html("<h3>prefixes:</h3>");
         $(".preview").html('<div class="schema-list-dialog-canvas rdf-kust-dialog-canvas"></div>');
         var links = $("<ul>").css("margin-left", "20px");
-        var skeleton = JSON.parse(data[event.target.id].skeleton);
+        var skeleton = JSON.parse(data[evt.target.id].skeleton);
```

Comment:
previous version worked in google chrome but not firefox, renaming event to evt fixed this problem

Reply:
If you only changed this line, you were lucky it worked at all? The function argument is called `evt`, and that didn't change?

Reply:
yes i was surprised as well, if chrome would have caught the error i would have seen this earlier